src/app.js

```diff
@@ -3,9 +3,13 @@ const app = express();
 const handlebars = require('express-handlebars');
 const router = require('./controllers/index');
```

Comment:
index is the entry point. You don't need to include it in the require statement.
should be 
```js
 require('./controllers');
```

Reply:
This require should happen after all the dependencies are required (requiring files should happen after requiring the modules)